Library/Styles/LoginStyles.swift

```diff
@@ -100,6 +100,17 @@ public let loginRootStackViewStyle =
         : .init(topBottom: Styles.grid(2), leftRight: Styles.grid(3))
 }
 
+public let facebookConfirmationStackViewStyle =
+  UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+    <> UIStackView.lens.layoutMargins %~~ { _, stack in
+      stack.traitCollection.isRegularRegular
+        ? .init(topBottom: Styles.grid(10), leftRight: Styles.grid(20))
+        : .init(
+          topBottom: stack.traitCollection.isVerticallyCompact ? Styles.grid(2) : Styles.grid(10),
+          leftRight: Styles.grid(3)
```

Comment:
Hey @bormind we want to keep all the login flow views consistent with margin padding. That means Login, ForgotPassword, SignUp, and Facebook views should all be using the same top margin padding (`Styles.grid(2)`) and thus share the `loginRootStackViewStyle`. Have you looked into the storyboard to see what else might be affecting the padding here? 

Also, if you are going to change landscape layouts for one view in a shared flow, then it is best to alter all views in this flow to be consistent.